src/compile/facet.ts

```diff
@@ -127,15 +127,7 @@ export class FacetModel extends Model {
       if (facet[channel]) {
         const axisSpec = facet[channel].axis;
         if (axisSpec !== false) {
-          let vlOnlyAxisProperties: VlOnlyAxisBase = {};
-          VL_ONLY_AXIS_PROPERTIES.forEach(function(property) {
-            if (config.facet.axis[property] !== undefined) {
-              vlOnlyAxisProperties[property] = config.facet.axis[property];
-            }
-          });
-
           const modelAxis = _axis[channel] = {
```

Comment:
Currently `VL_ONLY_AXIS_PROPERTIES` does not have `labelMaxLength`(which was the only property), why do we need to have it back? Do you want to add `VL_ONLY_AXIS_PROPERTIES` back as an empty object or add it back later if we have anything else in the future?